Katas/FizzBuzzTest/Changelog.md

```diff
@@ -0,0 +1,5 @@
+﻿# Changelog
+
+15/07/2019
+----------
+*New: test for case 1 and minimum passing code.
```

Comment:
it's nice to see you thinking in these terms - but you can just put this information into the git commits - that's what the commit messages are there for

Reply:
> ok great! I love that you've kept every step as a separate commit. Let me see exactly what your process was for working through this and you've done it perfectly.
> 
> In terms of the final solution you wrote, have you ever heard of object calisthenics?

No i havent heard of that i shall look into it thanks!